webpack/webpack.config.js

```diff
@@ -117,6 +130,38 @@ function createBundleAnalyzer() {
 	} );
 }
 
+function addYoastExternals( config, base, plugins, pluginVersionSlug ) {
```

Comment:
This works, but for 99% of the config that is duplicated here dynamically it's impossible to follow _why_. Only when you understand our particular setup with externals is it possible to understand this config. I'd rather see this solved differently. Without this, the confusing object `YoastExternalsMapping` is also not needed anymore. Please see if you can use the [dependency extraction webpack plugin](https://github.com/WordPress/gutenberg/tree/master/packages/dependency-extraction-webpack-plugin) for this.  If it doesn't support configuration yet, it might be easy to contribute that back so we don't have to fork the entire plugin.